.gitignore

```diff
@@ -7,3 +7,4 @@ __pycache__/
 *.egg-info/
 /cafebabel/static/uploads/
 /data/
+*.json
```

Comment:
It feels restrictive for the future to exclude any json file from the project

Reply:
Maybe we should have an `imports` folder to store that kind of files?